analyzers/src/SonarAnalyzer.CSharp/RspecStrings.resx

```diff
@@ -12553,6 +12553,33 @@
   <data name="S5042_Type" xml:space="preserve">
     <value>SECURITY_HOTSPOT</value>
   </data>
+  <data name="S5122_Category" xml:space="preserve">
+    <value>Minor Security Hotspot</value>
+  </data>
+  <data name="S5122_Description" xml:space="preserve">
+    <value>Having a permissive Cross-Origin Resource Sharing policy is security-sensitive. It has led in the past to the following vulnerabilities:</value>
```

Comment:
this description is a bit abrupt. Maybe there's something to improve in our tooling, to ensure we don't include sentences ending in `:`?

Reply:
I think this feedback is for the appsec team. @pierre-loup-tristant-sonarsource

Reply:
Can you elaborate @andrei-epure-sonarsource? Is it not correct to end the sentence with `:`?

Reply:
Hmm nevermind. This won't show up in SonarLint. This is the description that shows up in SonarLint AFAIK, but security hotspots don't show up there.

The problem I raised initially is that if a users sees in the IDE `It has led in the past to the following vulnerabilities:` without the actual list, it's weird.

Reply:
It looks like all the Security-Hotspots description are truncated in this file and only the first line appear.

> This is the description that shows up in SonarLint AFAIK, but security hotspots don't show up there
    
I think they do now: https://community.sonarsource.com/t/sonarlint-for-vscode-1-20-1-released-review-security-hotspots-in-the-ide/38162

Reply:
To be checked with SonarLint if they query the HTML from SonarQube/SC - @duncanp-sonar ?